partner_phonecall_schedule/models/res_partner.py

```diff
@@ -65,7 +66,13 @@ def _phonecall_available_domain(self):
             tz = timezone(self.env.context["tz"])
         except KeyError:
             tz = None
-        now = datetime.now(tz=tz)
+        now = self.env.context.get("now", datetime.now(tz=tz))
```

Comment:
Use `fields.Datetime.now()`

Reply:
Nope, I need a datetime object here.

Reply:
`fields.Datetime.from_string(fields.Datetime.now())` and you don't have to deal with tz.

Reply:
But that's exactly the same as `datetime.now()`, and the problem with tz is that hour fields are floats, and thus not auto-localized, so I need to localize them manually here.

Reply:
OK then